services/workflow.go

```diff
@@ -41,6 +52,87 @@ func (workflow Workflow) Call(params ...interface{}) (result interface{}, err er
 
 func ld_ld_sync(bytes []byte) error {
 	log.Println("Executing")
-	log.Println(string(bytes))
+
+	// Unmarshal the JSON byte array into the map
+	var jsonBody map[string]interface{}
+	err := json.Unmarshal(bytes, &jsonBody)
+	if err != nil {
+		slog.Error(err.Error())
+		return err
+	}
+
+	if jsonBody["LeadStatus"] == "Referred Out" && jsonBody["LeadSubstatus"] == "Pending Review" && jsonBody["LeadReferredTo"] == "The Capital Law Firm" {
+		// There should be more error handling so that when an error is raised we should try to send the request later, right now
+		// these messages are also offset
+		sendLeadToClf(jsonBody)
```

Comment:
Good comment here. Please go ahead and catch then log the error. So in production when things go wrong we can see it.

Reply:
Those errors are logged inside sendLeadToClf function. If there's an additional logic we're going to implement like I commented, yeah sure.
I mean currently if an error occurs inside the function, we just move on from the message after logging it.
It was probably a premature comment from my part before grasping the projects' requirements.

Reply:
Or I can just do this
```
if jsonBody["LeadStatus"] == "Referred Out" && jsonBody["LeadSubstatus"] == "Pending Review" && jsonBody["LeadReferredTo"] == "The Capital Law Firm" {
		err := sendLeadToClf(jsonBody)
		if err != nil {
			slog.Error("Error occurred while sending lead to CLF", "error", err)
			return err
		}
	}
```

Your call

Reply:
Just go ahead with the second one for now. I think it's good practice to produce the errors even to log, instead of leaving them hanging.